### PR TITLE
Add a simple doc on the convex poly constraints.

### PR DESCRIPTION
**Pull request by VeriTas-arch**
## Contents

- A simple note (line 582) on that polys can only be convex (see issue here #257 ), yet the doc content can be further discussed.
- Some minor changes on the doc to unify the style.

### CODE REVIEW

**Review by VeriTas-arch — COMMENTED** [review]
Most of them are minor changes.

File: pymunk/shapes.py [hunk 1]
```diff
@@ -578,14 +578,16 @@ def __init__(
     ) -> None:
         """Create a polygon.
 
-        A convex hull will be calculated from the vertexes automatically.
+        A convex hull will be calculated from the vertexes automatically. Note
+        that concave ones will be converted to a convex hull using the Quickhull
+        algorithm.
 
-        Adding a small radius will bevel the corners and can significantly
+        Add a small radius will bevel the corners and can significantly
```
Comment (viblo) [inline pymunk/shapes.py:585]: I think Adding is the correct way here. This is what the user would need pass in to this method, not what the method does.

**Review by viblo — CHANGES_REQUESTED** [review]
A minor comment

**Review by VeriTas-arch — COMMENTED** [review]
> I think Adding is the correct way here. This is what the user would need pass in to this method, not what the method does.


Well, it is in deed an omission, which is fixed already.

**Review by viblo — APPROVED** [review]

**Comment by viblo** [comment]
Thanks for the PR!

**Pull request merged**